packages/cli-repl/package.json

```diff
@@ -30,13 +30,20 @@
     "node": "^12.4.0"
   },
   "dependencies": {
+    "@mongosh/errors": "^0.0.1-alpha.13",
     "@mongosh/history": "^0.0.1-alpha.13",
     "@mongosh/i18n": "^0.0.1-alpha.13",
     "@mongosh/service-provider-server": "^0.0.1-alpha.13",
     "@mongosh/shell-api": "^0.0.1-alpha.13",
     "@mongosh/shell-evaluator": "^0.0.1-alpha.13",
+    "acorn": "^7.1.1",
+    "acorn-class-fields": "^0.3.2",
+    "acorn-numeric-separator": "^0.3.0",
+    "acorn-private-methods": "^0.3.1",
+    "acorn-static-class-features": "^0.2.1",
     "ansi-escape-sequences": "^5.1.2",
     "bson": "^4.0.4",
+    "is-recoverable-error": "^1.0.0",
```

Comment:
this should be @mongodb-js/is-recoverable-error right? Otherwise LGTM!

Reply:
it's not scoped, it just stayed as `is-recoverable-error`, but the package is here: https://github.com/mongodb-js/is-recoverable-error. Otherwise we have to go through a very painful process of deprecating it and creating a new package. Lots of our utils in mongodb-js are not scoped on github (for example [hadron-app-registry](https://www.npmjs.com/package/hadron-app-registry), or even [bson-transpilers](https://www.npmjs.com/package/bson-transpilers)), so I think it should be ok!

Reply:
fair enough! It's definitely not worth the pain. LGTM!

Reply:
hah all good!

yep, directly copy pasted, the only thing I added were tests! I have added the cases you mentioned, but if you have any other ones you can think of, let me know.